eg/Classes/WindowsVersion.py

```diff
@@ -0,0 +1,221 @@
+
+    def __gt__(self, other):
+        self.check_os_value(other)
+        if other == 'XP':
+            return self.THIS >self. WIN_XP32 and self.THIS >self. WIN_XP64
+        if other == '8':
+            return self.THIS >self. WIN_80 and self.THIS >self. WIN_81
+        return self.THIS > getattr(self, 'WIN_' + other.upper())
+
+    def __ge__(self, other):
+        self.check_os_value(other)
+        return self > other or self == other
+
+    def __lt__(self, other):
+        self.check_os_value(other)
+        if other == 'XP':
+            return self.THIS <self. WIN_XP32 and self.THIS <self. WIN_XP64
+        if other == '8':
+            return self.THIS <self. WIN_80 and self.THIS <self. WIN_81
+        return self.THIS < getattr(self, 'WIN_' + other.upper())
+
+    def __le__(self, other):
+        self.check_os_value(other)
+        return self < other or self == other
+
+    def __str__(self):
+        return "{0} {1}".format(
+            platform.system(),
+            platform.version()
+        )
+
+    def IsXP(self):
+        """
+        Checks if Windows version is:
+
+        Windows XP x86
+        Windows XP x64
+        Windows Server 2003
+        Windows 2003 R2
+        Windows Tablet PC
+        Windows Media Center Edition 2002
+        Windows Media Center Edition 2004
+        Windows Media Center Edition 2005
+
+        :return: Bool
+        """
+        return (
+            self.THIS == self. WIN_XP32 or
+            self.THIS == self. WIN_XP64
+        )
+
+    def IsVista(self):
+        """
+        Checks if Windows version is:
+
+        Windows Vista
+        Windows Server 2008
+
+        :return: Bool
+        """
+        return self.THIS == self. WIN_VISTA
+
+    def Is7(self):
+        """
+        Checks if Windows version is:
+
+        Windows 7
+        Windows Server 2008 R2
+
+        :return: Bool
+        """
+        return self.THIS == self. WIN_7
+
+    def Is8(self):
+        """
+        Checks if Windows version is:
+
+        Windows 8
+        Windows 8.1
+        Windows Server 2012
+        Windows 2012 R2
+        Windows RT
+        Windows RT 8.1
+
+        :return: Bool
+        """
+        return (
+            self.THIS == self. WIN_80 or
+            self.THIS == self. WIN_81
+        )
+
+    def Is80(self):
+        """
+        Checks if Windows version is:
+
+        Windows 8
+        Windows Server 2012
+        Windows RT
+
+        :return: Bool
+        """
+        return self.THIS == self. WIN_80
+
+    def Is81(self):
+        """
+        Checks if Windows version is:
+
+        Windows 8.1
+        Windows 2012 R2
+        Windows RT 8.1
+
+        :return: Bool
+        """
+        return self.THIS == self. WIN_81
+
+    def Is10(self):
+        """
+        Checks if Windows version is:
+
+        Windows 10
+        Windows Server 2016
+
+        :return: Bool
+        """
+        return self.THIS == self. WIN_10
```

Comment:
Formatting:

    return self.THIS == self.WIN_10

Construct WindowsVersion

    WindowsVersion = WindowsVersion()